### PR TITLE
Feat: add message_reaction event

### PR DESCRIPTION
**Pull request by wictornogueira**
Emits an event whenever a reaction gets updated/created

### CODE REVIEW

File: src/Client.js [hunk 1]
```diff
@@ -439,6 +439,27 @@ class Client extends EventEmitter {
             this.emit(Events.INCOMING_CALL, cll);
         });
 
+        await page.exposeFunction('onReaction', (reactions) => {
+            for (const reaction of reactions) {
+                /**
+                 * Emitted when a reaction is sent, received, updated or removed
+                 * @event Client#incoming_call
```
Comment (PurpShell) [inline src/Client.js:446]:
```suggestion
                 * @event Client#message_reaction
```

**Review by purpshell — APPROVED** [review]
ok I can't roast this wictor, I approve but I found one mistake hehheh

**Review by tuyuribr — CHANGES_REQUESTED** [review]
Hey @wictornogueira Really cool event, can you add an example on example.js ?

**Comment by PurpShell** [comment]
@tuyuribr we plan to remove that file after the coverage of the [guide](https://wwebjs.dev/guide).

@wictornogueira merge conflicts

**Review by tuyuribr — APPROVED** [review]

**Pull request merged**